cpp/lib/ome/test/main.cpp

```diff
@@ -45,3 +45,11 @@ main (int   argc,
   testing::InitGoogleTest(&argc, argv);
   return RUN_ALL_TESTS();
 }
+
+bool
+verbose ()
+{
+  static bool v(getenv("BIOFORMATS_TEST_VERBOSE")
+                && std::string(getenv("BIOFORMATS_TEST_VERBOSE")) == "true");
```

Comment:
I assume there's other code elsewhere that is equally picky about "true"'s case, etc., so this is good in context.